src/main/resources/messages.yml

```diff
@@ -338,6 +338,7 @@ other:
     send-on-join: true
 
   mob-head-drop-name: '%mob_name%''s head'
+  no-drop-in-chunk: '&6You have reached &amaximum&6 kill count in this chunk. &4Please come back after a while.'
```

Comment:
this doesn't follow LM's color scheme - instead we should probably use this:
`&7Your levelled mob kill count has reached the maximum for this chunk. You will no longer receive levelled drops from these mobs. Please come back after a while.`